dali/test/python/test_pipeline_multichannel.py

```diff
@@ -166,8 +166,7 @@ def check_multichannel_synth_data_vs_numpy(tested_operator, device, batch_size,
     eii2 = RandomDataIterator(batch_size, shape=shape)
     compare_pipelines(MultichannelSynthPipeline(device, batch_size, "HWC", iter(eii1), tested_operator=tested_operator),
                       MultichannelSynthPythonOpPipeline(get_numpy_func(tested_operator), batch_size, "HWC", iter(eii2)),
-                      batch_size=batch_size, N_iterations=10,
-                      eps = 0.2)
+                      batch_size=batch_size, N_iterations=3, eps = 0.2)
```

Comment:
Is it OK to reduce number of iterations for the test, which has significantly broader tolerance? Maybe we should ask the question, why this tolerance is broader?

Reply:
I believe it is OK. Some time ago we didn't think too much about the number of iterations. 10 is just an arbitrary number, I think. At some point, we decided that doing 3 iterations was enough (more than one iteration is necessary, since there are some issues when the operator doesn't clear the state properly between iterations). If we want to test more, we normally write a different test (different arguments, different kind of data, etc). Increasing the number of iterations is not equivalent of better coverage.

Regarding why the tolerance is broader, I don't have a good answer for it. This tests a whole pipeline with several operator chained, using randomly generated data, against a reference implemented in OpenCV. I assume the bigger eps is due to the data being pure noise, some subtle differences in the implementation might result in a larger error (interpolation, for instance). In line 236 we have the same test but with actual images, and the eps is much lower there.